ophyd/controls/signal.py

```diff
@@ -475,6 +475,10 @@ def get(self, **kwargs):
         return [signal.get(**kwargs)
                 for signal in self._signals]
 
+    @property
+    def signals(self):
+        return self._signals
```

Comment:
As `_signals` is mutable (a list), you'd want to duplicate it here prior to returning it.  There are a few other instances in the PR similar to this.